lambda/Makefile

```diff
@@ -0,0 +1,43 @@
+LOCAL_ENDPOINT_HOST    ?=localhost
+LOCAL_ENDPOINT         =--endpoint-url=http://${LOCAL_ENDPOINT_HOST}
+SNS_ENDPOINT           =${LOCAL_ENDPOINT}:4575
+ES_ENDPOINT            =${LOCAL_ENDPOINT}:4578
+LAMBDA_ENDPOINT        =${LOCAL_ENDPOINT}:4574
+PROJ_ES_DOMAIN         =records
+PROJ_SNS_TOPIC         =data-update
+PROJ_AWS_REGION        =us-east-1
+PROJ_AWS_ACCESS_KEY_ID =999999
+PROJ_AWS_SECRET_KEY    =999999
+PROJ_ENV_VARS          =AWS_REGION=${PROJ_AWS_REGION} AWS_ACCESS_KEY_ID=${PROJ_AWS_ACCESS_KEY_ID} AWS_SECRET_ACCESS_KEY=${PROJ_AWS_SECRET_KEY}
+
+default: package upload subscribe
+
+package:
+	GOOS=linux go build -o main
+	zip lambda.zip main
+
+upload:
+	$(eval FUNCTION=$(shell aws $(LAMBDA_ENDPOINT) lambda list-functions | jq '.Functions[0].FunctionName // ""'))
+	@if [[ $(FUNCTION) != "" ]]; \
+	  then echo "$(FUNCTION) function found"; \
```

Comment:
We may want to leave a message here that this script is not replacing what already exists.

Reply:
@aaron-collier updated this PR after #3 was merged. He and @tingulfsen and I reviewed this and we weren't sure what this comment meant. Going to merge the PR -- if you wouldn't mind writing this up a new issue, we can revisit this later.